autokeras/tasks/image.py

```diff
@@ -23,7 +23,8 @@ class ImageClassifier(SupervisedImagePipeline):
     """AutoKeras image classification class.
 
     # Arguments
-        num_classes: Int. Defaults to None. If None, it will infer from the data.
+        num_classes: Int. Defaults to None. If None, it will be inferred from the
```

Comment:
I just realized there may be more of this grammar error. Would you please use a global search to find them all and change them?